components/teachers/styles.js

```diff
@@ -16,8 +16,19 @@ function styles(theme) {
             "& input": {
                 color: red[400],
                 borderBottom: "1px solid " + red[400],
-                fontWeight: "bold"
-            }
+                fontWeight: "bold",
+                textAlign: "center"
+            },
+            '& strong': {
+                color: red[400],
+            },
+            '& p': {
+                color: red[400],
+            },
+            color: red[400],
+        },
+        teacherWarning: {
+            border: "2px solid" + red[400],
```

Comment:
Again, consistency with the trailing commas. Just a little thing that makes the code look more professional.